src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -0,0 +1,19 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+       xmlns:aop="http://www.springframework.org/schema/aop" xmlns:tx="http://www.springframework.org/schema/tx"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                           http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
+                           http://www.springframework.org/schema/tx
+                           http://www.springframework.org/schema/tx/spring-tx-2.0.xsd
+                           http://www.springframework.org/schema/aop
+                           http://www.springframework.org/schema/aop/spring-aop-2.0.xsd">
+
+    <bean id="arModuleConfiguration" parent="arModuleConfiguration-parentBean">
```

Comment:
I think we'll want to define a customized <bean id="cu-ar" parent="arModuleConfiguration-parentBean">. Take a look at cu-spring-coa.xml or other locally customized modules.